### DISCUSSION TITLE
But what are the 24 Cardinal host-exposed parameters???

### DISCUSSION THREAD

**Discussion by Vadym-beep** [DISTRHO/Cardinal#470] [Q&A] [answered]
I don't understand how the host parameter module works, I loaded up Cardinal in FL Studio as a VST3 plugin and I don't know how to control the parameters, I need to control them without external hardware (MIDI Controllers) because I don't have any and not planning to buy it. How do I control them and what are these parameters.

**Comment by falkTX** [comment, marked as answer]
FLStudio is a weird one as it does not list parameters exposed by a plugin and expect users to move a knob matching the parameter to automate.
You can just click and drag any of the "Host Params" 24 cables, drop it anywhere without a connection, it is just enough to register an event to hosts like FLStudio to recognize it as the "last changed parameter".
